src/features/create-quiz/create-quiz-question-list/components/question-item.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Card } from "primereact/card";
+import { Button } from "primereact/button";
+
+import { observer } from "mobx-react";
+interface QuestionItemProps {
+  questionName: string;
+  handleClick: any;
+  answers: number;
```

Comment:
I suppose that the prop `answers` represents the number of answers, so a more suggestive naming could be something like `answersCount` or `numberOfAnswers` etc. to make it more clear.